server/gateways/cloudflare.js

```diff
@@ -0,0 +1,55 @@
+import config from 'config';
```

Comment:
Why not put this library in `lib` instead of `gateways`?

Reply:
`gateways` seemed to me like the place where we wrap third-party libraries. What's the purpose of this folder if that's not for that?

Reply:
Then I would drop `gateways` and have everything in `lib`, this is where most wrappers are already.